### PR TITLE
Adding information for using TinyMCE editor and FormattedTextareaType with prestashop UI kit and twig form

### PR DESCRIPTION
**Pull request by Rizzen59**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x / 8.x
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Comment by Rizzen59** [comment]
I had to seach why my editor was not displayed (I found the solution by looking on Maintenance twig from) so to avoid other developers to loose time. I propose to improove documentation.

File: development/components/form/types-reference/formatted-textarea.md [hunk 1]
```diff
@@ -15,7 +15,17 @@ Enables TinyMCE text editor on TextareaType.
 
 ## Required Javascript components
     
-None.
+To use with twig and new prestashop admin UI Kit. You have to init the component :
```
Comment (matthieu-rolland) [inline development/components/form/types-reference/formatted-textarea.md:18]: To make it work in the PrestaShop UI Kit, you need to call the `TinyMCEEditor` js component :

**Review by matthieu-rolland — COMMENTED** [review]
Thank you for your PR @Rizzen59 👍

File: development/components/form/types-reference/formatted-textarea.md (hunk 1 above)
Comment (matthieu-rolland) [inline development/components/form/types-reference/formatted-textarea.md]:
```suggestion
To make it work in the PrestaShop UI Kit, you need to initialize the `TinyMCEEditor` js component:
```

Comment (saulaski) [inline development/components/form/types-reference/formatted-textarea.md]: Alternative wording:
```suggestion
In order to use TinyMCE editor in the PrestaShop UI Kit, you need to initialize its javascript component once the page is loaded:
```

**Review by saulaski — CHANGES_REQUESTED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
hi @Rizzen59 !

you need to commit @saulaski 's wording and the PR is good to go 👍

**Comment by Rizzen59** [comment]
Thanks, i will make the change

**Review by saulaski — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @Rizzen59 

Could you create the same PR on 8.x branch ?

**Pull request merged**